tldr.py

```diff
@@ -120,7 +118,7 @@ def have_recent_cache(command, platform):
         return False
 
 
-def get_page_for_platform(command, platform):
+def get_page_for_platform(command, platform, remote):
```

Comment:
You have to pass the remote here too inside the get_page call from where this function is called.